bot/handlers/administration/list_keys_handler.py

```diff
@@ -0,0 +1,37 @@
+import logging
+from typing import List
+
+from aiogram.types import Message
+
+from bot.database.database_manager import DatabaseManager
+from bot.handlers.bot_message_handler import BotMessageHandler
+from bot.responses.administration.list_keys_handler_responses import (
+    create_subscription_keys_response,
+    get_log_subscription_keys_empty_message,
+    get_log_subscription_keys_sent_message,
+    get_subscription_keys_empty_message,
+)
+
+
+class ListKeysHandler(BotMessageHandler):
+    def get_commands(self) -> List[str]:
+        return ["listkey", "lk"]
+
+    async def is_any_validation_failed(self, message: Message) -> bool:
+        return False
```

Comment:
no zdecydowanie powinieneś to zrobić tak żeby ta lista była domyślnie pusta żeby nie trzaskać tego boilerplate wszędzie albo z `False`, albo z pustą listą